README.md

```diff
@@ -38,6 +51,36 @@ scopelint fmt
 scopelint check
 ```
 
+## Scripts
+
+ * `script/Deploy.s.sol` - Deploys the GitcoinGovernor contract
+ * `script/Propose.s.sol` - Submits a proposal to the existing Gitcoin Governor Alpha proposing migration to the GitcoinGovernor. Must be executed by someone with sufficient GTC delegation.
+
+ To test these scripts locally, start a local fork with anvil:
+
+ ```bash
+ anvil --fork-url YOUR_RPC_URL --fork-block-number 15980096
+ ```
+
+ Then execute the deploy script.
+
+ _NOTE_: You must populate the `DEPLOYER_PRIVATE_KEY` in your `.env` file for this to work.
+
+ ```bash
+ forge script script/Deploy.s.sol --tc DeployScript --rpc-url http://localhost:8545 --broadcast
```

Comment:
When I run this using a private key that has sufficient mainnet balance, it errors with `No sender account configured: $ETH_FROM`. Guessing this doesn't happen to you? If so, I think it may be an issue with some old dapptools keystore I have setup and I can investigate without blocking this PR

Reply:
Just confirmed again I do not have this issue with latest version of Foundry.

Reply:
Marking resolved, I'll investigate this myself separately